### DISCUSSION TITLE
SimBrief Blocked Fuel calculation - Excessive ZFW load

### DISCUSSION THREAD

**Discussion by SagiEran** [flybywiresim/aircraft#6244] [Q&A] [answered]
Hi Guys,

I'm using SimBrief as a planning tool and the calculated blocked fuel is too high.
I'm ending up with non realistic amount of fuel after flight. some times as high as 35% of total ZFW (after deduction of ALTR and the Extra mandatory 30 minutes flight time). taxi is set at 200kg for taxing.
Can you share your simbrief A320NX profile and Fuel settings ? (using Dev 0.7.3 latest).

Often have a burn rate of : 900 per engine per hour at Cruise level

Thanks, Eran

**Comment by CptBayer** [comment]
Hi,


did you use the new profile from Simbrief for the A320N? The profile is important cause for example the A 320 ceo has a fuel burn of about 1.400-1-500 kg/h. The Neo between 1.000 and 1.100 kg/h (round about). That is one reason for a high block fuel. The engine model is not yet the finished one, still in development, but till now the values a really good. In my profile I use all the values from Simbrief only the fuelfactor I chosse is P06 (plus 6 %). That does fit with the fuel consumption quite good... I am only using dev version, not stable 0.73.

![grafik](https://user-images.githubusercontent.com/76531893/143424408-2b59d40b-93cd-42b5-8e22-164e12b427a7.png)

greets

**Comment by speedwaystar** [comment, marked as answer]
here is a more recent profile, specifically designed for the new W/B system which was just introduced to the Dev build.
https://www.simbrief.com/system/dispatch.php?sharefleet=337364_1632489487665

important changes: max passengers 174, and OEW (empty weight) of 42500 kg, fuel factor P10.

for more info see the W/B update testing instructions at https://github.com/flybywiresim/a32nx/pull/5895

**Reply by SagiEran** [reply]
Thanks,

this looks more realistic in perf. I will try that configuration. thanks !
